src/component/NMRium.tsx

```diff
@@ -360,13 +361,16 @@ function InnerNMRium({
                                   <div css={viewerContainerStyle}>
                                     <KeysListenerTracker />
                                     <div
+                                      id="nmrium-viewer"
                                       data-test-id="viewer"
                                       ref={viewerRef}
                                       style={{
                                         width: '100%',
                                         height: '100%',
+                                        position: 'relative',
                                       }}
                                     >
+                                      <FloatMoleculeStructures />
```

Comment:
It seems you wanted to move the rendering of FloatMoleculeStructures to this place, but it is still rendered a second time elsewhere in the DOM (you can search for it in the react dev tools, the other render has a size of 0x0)

Reply:
@hamed-musallam Could you explain why you moved it ? Because moving the molecule was already working correctly so we expected that resizing would work as well.